apps/server/browser/styles.css

```
@@ -26,41 +26,130 @@ body {
-  flex-direction: column;
+.server-button:disabled {
+  cursor: not-allowed;
+  opacity: 0.5;
+}
+
+.server-button svg,
+.server-input-group svg {
+  width: 16px;
+  height: 16px;
+  flex: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)